generation/hermetic_library_generation.sh

```diff
@@ -102,7 +109,13 @@ docker run \
   --repository-path="${workspace_name}"
 # commit the change to the pull request.
 git add java-* pom.xml gapic-libraries-bom/pom.xml versions.txt "${generation_config}"
```

Comment:
I might missed this from last PR, why we need to add `versions.txt` and `generation_config`? I don't think our scripts are touching those files.

Reply:
I removed `generation_config`.

A new client generation will change `versions.txt` (adding new entry).